gcloud-java-dns/src/main/java/com/google/cloud/dns/testing/LocalDnsHelper.java

```diff
@@ -112,6 +119,15 @@
   private static final ScheduledExecutorService EXECUTORS =
       Executors.newScheduledThreadPool(2, Executors.defaultThreadFactory());
   private static final String PROJECT_ID = "dummyprojectid";
+  private static final String responseBoundary = "____THIS_IS_HELPERS_BOUNDARY____";
+  private static final String responseSeparator = new StringBuilder("--")
```

Comment:
Nit, because all string values are constants simple concatenation would be more efficient:
```java
private static final String responseSeparator = "--" + responseBoundary + "\r\n";
```
as the compiler will make it one string.